sklearn/manifold/t_sne.py

```diff
@@ -567,10 +568,11 @@ class TSNE(BaseEstimator):
         the distance between them. The default is "euclidean" which is
         interpreted as squared euclidean distance.
 
-    init : string, optional (default: "random")
-        Initialization of embedding. Possible options are 'random' and 'pca'.
-        PCA initialization cannot be used with precomputed distances and is
-        usually more globally stable than random initialization.
+    init : string or NumPy array, optional (default: "random")
```

Comment:
Should you use the more consistent `numpy array` instead of `NumPy` array? Also it would be nice to specify as `numpy array of shape [n_samples, n_components]`.